reftools/getphotpars.py

```diff
@@ -228,7 +228,7 @@ def _get_row(self, obsmode, ext):
                 f'Obsmode {obsmode} appears multiple times in '
                 f'{self.imphttab_name} extension {ext}')
 
-        return self.imphttab_fits[ext].data[w]
+        return self.imphttab_fits[ext].data[w[0][0]]
```

Comment:
There is a bunch of checks above that ensure `len(w[0]) == 1`, so I think it is okay to just use `w[0][0]` here.